src/main/java/zbot/StorageManager.java

```diff
@@ -0,0 +1,145 @@
+package zbot;
+
+import zbot.exceptions.ZBOTFileNotFoundException;
+
+import zbot.tasks.ToDoTask;
+import zbot.tasks.EventTask;
+import zbot.tasks.DeadlineTask;
+import zbot.tasks.Task;
+import zbot.tasks.TaskList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+
+import java.util.Scanner;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
coding standard: consider cleaning up this list of imports to maintain alphabetical order + logical grouping:

```suggestion
import zbot.exceptions.ZBOTFileNotFoundException;

import zbot.tasks.DeadlineTask;
import zbot.tasks.EventTask;
import zbot.tasks.Task;
import zbot.tasks.TaskList;
import zbot.tasks.ToDoTask;

import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileWriter;
import java.io.IOException;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;
```

Reply:
thanks for the input! implemented it:)